mc-http-server does not start

We are facing an issue in all our custom apps, we are pulling this image

FROM eu.gcr.io/ct-images/mc-http-server:v11.2.0

And now the log show us, that a module is missing.
Can you help us with that?

Let me know if need more logs or more information for us.

<img width="1050" alt="Captura de Pantalla 2020-05-08 a la(s) 12 25 48" src="https://user-images.githubusercontent.com/29213153/81430224-17f96e80-912d-11ea-83f0-b84d73ec1334.png">

Complete error 

Error: Cannot find module '@babel/runtime-corejs3/helpers/interopRequireDefault'
Require stack:
- /app/node_modules/@commercetools-frontend/mc-html-template/build/compile-html.js
- /app/node_modules/@commercetools-frontend/mc-html-template/build/index.js
- /app/bin/start-mc.js
at Function.Module._resolveFilename (internal/modules/cjs/loader.js:962:15) 
at Function.Module._load (internal/modules/cjs/loader.js:838:27)
at Module.require (internal/modules/cjs/loader.js:1022:19)
at require (internal/modules/cjs/helpers.js:72:18)
at Object.<anonymous> (/app/node_modules/@commercetools-frontend/mc-html-template/build/compile-html.js:3:30)
at Module._compile (internal/modules/cjs/loader.js:1118:30)
at Object.Module._extensions..js (internal/modules/cjs/loader.js:1138:10)
at Module.load (internal/modules/cjs/loader.js:982:32)
at Function.Module._load (internal/modules/cjs/loader.js:875:14)
at Module.require (internal/modules/cjs/loader.js:1022:19) {
code: 'MODULE_NOT_FOUND',
requireStack: [
'/app/node_modules/@commercetools-frontend/mc-html-template/build/compile-html.js',
'/app/node_modules/@commercetools-frontend/mc-html-template/build/index.js',
'/app/bin/start-mc.js'
]
}
internal/modules/cjs/loader.js:965
throw err;
^

I can confirm that I can reproduce the error with the `v16.8.0` image

```
docker run --rm -it eu.gcr.io/ct-images/mc-http-server 
internal/modules/cjs/loader.js:965
  throw err;
  ^

Error: Cannot find module '@babel/runtime-corejs3/helpers/interopRequireDefault'
Require stack:
- /app/node_modules/@commercetools-frontend/mc-html-template/build/compile-html.js
- /app/node_modules/@commercetools-frontend/mc-html-template/build/index.js
- /app/bin/start-mc.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:962:15)
    at Function.Module._load (internal/modules/cjs/loader.js:838:27)
    at Module.require (internal/modules/cjs/loader.js:1022:19)
    at require (internal/modules/cjs/helpers.js:72:18)
    at Object.<anonymous> (/app/node_modules/@commercetools-frontend/mc-html-template/build/compile-html.js:3:30)
    at Module._compile (internal/modules/cjs/loader.js:1118:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1138:10)
    at Module.load (internal/modules/cjs/loader.js:982:32)
    at Function.Module._load (internal/modules/cjs/loader.js:875:14)
    at Module.require (internal/modules/cjs/loader.js:1022:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/app/node_modules/@commercetools-frontend/mc-html-template/build/compile-html.js',
    '/app/node_modules/@commercetools-frontend/mc-html-template/build/index.js',
    '/app/bin/start-mc.js'
  ]
}
```

We'll look into it. Thanks again for reporting the issue and apologies for the inconvenience.

--- a/.changeset/shiny-colts-eat.md
+++ b/.changeset/shiny-colts-eat.md
@@ -1,0 +1,6 @@
+---
+'@commercetools-frontend/mc-html-template': patch
+'@commercetools-frontend/mc-scripts': patch
+---
+
+Missing `@babel/runtime-corejs3` dependency, causing the docker image of the `mc-http-server` to fail.

--- a/packages/mc-html-template/package.json
+++ b/packages/mc-html-template/package.json
@@ -37,6 +37,7 @@
   },
   "dependencies": {
     "@babel/runtime": "7.9.6",
+    "@babel/runtime-corejs3": "7.9.6",
     "core-js": "3.6.5",
     "node-fetch": "2.6.0",
     "serialize-javascript": "3.0.0",

--- a/packages/mc-scripts/package.json
+++ b/packages/mc-scripts/package.json
@@ -50,6 +50,7 @@
   },
   "dependencies": {
     "@babel/runtime": "7.9.6",
+    "@babel/runtime-corejs3": "7.9.6",
     "@commercetools-frontend/assets": "16.8.0",
     "@commercetools-frontend/babel-preset-mc-app": "16.8.0",
     "@commercetools-frontend/mc-dev-authentication": "16.8.0",

